volume/validate.go

```diff
@@ -7,8 +7,6 @@ import (
 	"github.com/pkg/errors"
 )
 
-var errBindNotExist = errors.New("bind source path does not exist")
-
```

Comment:
Instead of repeating the error message in a few places I think this `var` should be replaced with:

```go
func newErrBindSourceDoesNotExist(path string) {
    return errors.Errorf("bind mount source path does not exist: %s", path)
}
```

Reply:
That looks inline with the current implementation of other errors. 
Are you happy with this format:

`bind mount source path does not exist: %s`

or my version of the format is acceptable:

`bind mount source path: '%s' does not exist'

Reply:
I don't have a strong preference. Either is ok with me.

Reply:
I think I'll go with your format version - I hate quotes too! :smile: - with a slight change to the method name to match other methods naming convention.